### ISSUE TITLE
bad-continuation false alarm and failure to disable

### ISSUE THREAD

**Issue by kayhayen** [PyCQA/pylint#2304]
### Steps to reproduce
1.
I have this code piece:

```
        emit(
            """\
SET_CURRENT_EXCEPTION( exception_preserved_type_%(preserver_id)d, \
exception_preserved_value_%(preserver_id)d, \
exception_preserved_tb_%(preserver_id)d );""" % {
                "preserver_id" : preserver_id,
            }
        )
```

I think it wants the hanging "}" brace to align one level shorter, which is false.

2.

With that false alarm in place, --disable=bad-continuation on the command line has no effect. I cannot disable the warning until the bug is fixed.

3.

### Current behavior

Reports:

> nuitka\codegen\FrameCodes.py:446 C0330 bad-continuation  Wrong hanging indentation.
>             }
> |   |       ^

### Expected behavior

No report, and allow --disable on that error.

### pylint --version output

c/Python36_32/python -m pylint --version
__main__.py 2.0.0
astroid 2.0.0.dev4
Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 16:07:46) [MSC v.1900 32 bit (Intel)]

**Comment by PCManticore** [comment]
Thanks for the report!

Yes, it seems that it wants the indentation to be one level shorter, which I agree does not make sense here.
Also confirming that we cannot disable `bad-continuation` here.

**Comment by mthuurne** [comment]
I also ran into the problem that `bad-continuation` sometimes cannot be suppressed. I'll add my test case here, as it's a bit shorter than the one from the original report.

In `pylintrc`:
```
[MESSAGES CONTROL]
disable=bad-continuation

[FORMAT]
indent-after-paren=9999
```

Code to check:
```python
print(
    1,
    2,
    3
    )
```

An interesting effect is that the spaces set by `indent-after-paren` are printed as part of the error message (as spaces). I don't know if that's related to the bug, but it doesn't seem right.

**Comment by brycepg** [comment]
@mthuurne I fixed failure to disable `bad-continuation` near the end of the file (after ast nodes end but not tokens) in pylint `2.1`

**Comment by mthuurne** [comment]
Thanks!

**Comment by Pierre-Sassoulas** [comment]
`bad-continuation` has been removed in #3571, black or another formatter can help you with this better than Pylint

**Comment by kayhayen** [comment]
I agree @Pierre-Sassoulas
